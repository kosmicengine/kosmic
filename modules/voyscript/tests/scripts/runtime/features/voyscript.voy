func test():
	var voyscr: = VoyScript.new()
	voyscr.source_code = '''
extends Resource

func test() -> void:
	prints("Outer")
	var inner = InnerClass.new()

class InnerClass:
	func _init() -> void:
		prints("Inner")
'''
	@warning_ignore("return_value_discarded")
	voyscr.reload()

	var inst = voyscr.new()

	@warning_ignore("unsafe_method_access")
	inst.test()
