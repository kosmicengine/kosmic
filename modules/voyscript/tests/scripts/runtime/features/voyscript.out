VOYTEST_OK
Outer
Inner
